### ISSUE TITLE
Missing Auto Infusion Matrix Recipes

### ISSUE THREAD

**Issue by Electrolyte220** [Divine-Journey-2/Divine-Journey-2#702]
### Modpack version

2.15.0

### Description

Some recipes from the Thaumcraft Infusion Matrix are not available in the Auto Infusion Matrix. 
I looked through the recipes and the ones I found are: 
- Brazier
- Hecate's Visage
- Hood of the Adept
- Robe of the Adept
- Silver Candelabra
- Void Siphon
- Gyroscope Upgrade
- Aconitum Seeds
- Triskelion Amulet
- Girdle of the Dryads
- Dragon's Blood Sapling
- Wormwood Seeds
- Iron Candelabra
- Belladonna Seeds
- Golden Candelabra
- Leggings of the Adept
- Harness Base

### Steps to reproduce

1. Look at recipe for any item in the list above in JEI.
2. See that it can only be made in the regular infusion matrix.

### Expected behavior

I would like all (or most) of the infusion recipes from the list above to work in the auto infusion matrix. For items like the Bewitchment seeds or candelabras for example, they probably don't need an auto infusion matrix recipe since the player won't have access to the auto infusion matrix at that point in progression anyways.

**Comment by Electrolyte220** [comment]
I guess the only recipe I truly care about being added is the Brazier, since that is needed for progression, and I didn't feel like setting up an entire infusion setup just to tear it down after crafting the Brazier would be worth it. I get why the other items were excluded, since as you said, they are one time crafts and most players will avoid them since they aren't required for progression.

**Comment by King65868** [comment]
Brasier is in bewitchment, pre AIM

**Comment by Totox00** [comment]
Yeah but it's not required for making the AIM, and I would assume rushing the AIM is fairly common, so adding the recipe for the brazier at least would make sense

**Comment by JetMage28** [comment]
i believe that the triskelion amulet should be added to the AIM, because its used for the mob loot fab, and that appears significantly after AIM

**Comment by KleinGrrmpf** [comment]
> Brasier is in bewitchment, pre AIM

I would argue that even though advanced Thaumcraft is the chapter after Bewitchment, they are still at the same time in progression because you need both of them to unlock Arcana and Lightningcraft

**Comment by King65868** [comment]
> > Brasier is in bewitchment, pre AIM
> 
> I would argue that even though advanced Thaumcraft is the chapter after Bewitchment, they are still at the same time in progression because you need both of them to unlock Arcana and Lightningcraft
That is true, maybe it will be added to AIM then
